app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    names = command.split(" ")
```

Comment:
To improve efficiency, avoid calling `command.split(" ")` multiple times. Store the result in a variable and reuse it.